curl/inc/leatherman/curl/client.hpp

```diff
@@ -164,6 +164,13 @@ namespace leatherman { namespace curl {
          */
         void set_client_cert(std::string const& client_cert, std::string const& client_key);
 
+        /**
+         * Set and limit what protocols curl will support
+         * @param client_protocols bitmask of CURLPROTO_*
+         *        (see more: http://curl.haxx.se/libcurl/c/CURLOPT_PROTOCOLS.html)
+         */
+        void set_supported_protocols(long client_protocols);
```

Comment:
Yeah, not really sure what to say about this. The function itself is expecting them to be long values, where they are in some form `CURLPROTO_SOMETHING` :(